apps/client/src/app/core/notification/prompt-dialog/prompt-dialog.html

```diff
@@ -0,0 +1,21 @@
+@if (title) {
+  <div mat-dialog-title [innerHTML]="title"></div>
+}
+
+<div mat-dialog-content style="padding: 20px">
```

Comment:
Please use `class="py-3"` instead of `style="padding: 20px"` to align with other dialogs